### ISSUE TITLE
Use the pretrained model of ResNet50 on ImageNet, the val acc is 4% less than reported.

### ISSUE THREAD

**Issue by liu-zhenhua** [pytorch/vision#1944]
As the title mentioned, anyone have met the same issue?Thx!

**Comment by fmassa** [comment]
Hi,

Did you put the model in `.eval()` mode?
ResNet has batch normalization, which during training changes the statistics of the output and can lead to different reported results.

**Comment by liu-zhenhua** [comment]
> Hi,
> 
> Did you put the model in `.eval()` mode?
> ResNet has batch normalization, which during training changes the statistics of the output and can lead to different reported results.

Thanks for the reply. I have checked the code and I did put the model in '.eval' model. I forgot to mention that I resize the images to 256*256 in local. Do you think this may has an influence on the result? Thanks again.

**Comment by kentaroy47** [comment]
@liu-zhenhua 
the convolution filters and average pooling will act different if you choose a different size than the trained resolution (224x224).
Can you check if the accuracy change if you resize to 224x224?

**Comment by fmassa** [comment]
@liu-zhenhua Having the images be in 256x256 should not make the final val accuracy to be of only 4%, although it can have some impact.

I suspect it's an user error somewhere, but without more information it is impossible for us to find what the issue is.

Can you use the evaluation code from `references/classification` to test the pre-trained resnet50 model on imagenet?

**Comment by fmassa** [comment]
Closing following inactivity. If you still face the same issues please let us know.
